freezeyt/freezer.py

```diff
@@ -384,6 +395,20 @@ def add_static_task(
             task.asyncio_task = asyncio_create_task(coroutine, name=task.path)
         return task
 
+    def add_file_task(
```

Comment:
Dont we consolidate this func and `add_static_task` to just `add_task` or it was another PR?

Reply:
In another PR, when middleware serves the static files.